app/assets/stylesheets/my_css.css

```diff
@@ -0,0 +1,19 @@
+body {
+  padding-top: 50px;
+}
+
+.my_header {
+  background-color: #502840;
+  /*margin-top: 50px;*/
+  color: white;
+  border-radius: 0;
+}
+
+.my_card {
+  background-color: #344B5A;
+  color: white;
+}
+
+.bottom_shadow {
+  box-shadow: 0 0 3px 2px grey;
+}
```

Comment:
Prefer to maintain separate css files for different pages, for clearer organization. As a guide, you can name the css file after the controller action.